frontend/www/js/omegaup/time.test.ts

```diff
@@ -1,5 +1,41 @@
 import * as time from './time';
 
+describe('getDifferenceInCalendarYears', () => {
+  // Setting an specific datetime to avoid flakiness in a leap-year
+  const now = new Date(0).getDate();
```

Comment:
```suggestion
  const now = new Date('2000-01-01T00:00:00Z').getTime();
```

it's best to use a human-readable date to ensure that the tests are correct. Also, [`getDate` returns the day of the month](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate) (not what we want), whereas [`getTime` returns the timestamp from the epoch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime) (what `Date.now()` returns).